README.md

```diff
@@ -1 +1,16 @@
-# awa-scraper
\ No newline at end of file
+# awa-scraper
+
+## Develop
+```console
+$ python setup.py develop
+$ python -m awa
+2021-08-24T00:38:28.130Z INFO:root:Doing something
+```
+
+## Lint
+Run `black` against everything
+```console
+$ pip install black
+$ find . -iname "*.py" -exec black {} \;
```

Comment:
You _should_ be able to run `black` recursively just by using an invocation like `black .`